tasks/main.yml

```diff
@@ -0,0 +1,127 @@
+---
+- name: Gather needed facts
+  setup:
+    gather_subset:
+      - '!all'
+      - '!min'
+      - distribution_major_version
+  when: ansible_facts.distribution_major_version is not defined
+
+- name: System check
+  fail:
+    msg: >-
```

Comment:
I suggest writing this in multi-line for better readability, like  this:

```
    msg:
      - Only Red Hat Enterprise Linux >= 8.1 is supported.
      - System: {{ ansible_facts.os_family }}
      - Version: {{ ansible_facts.distribution_version }}
```

Otherwise, it is printed as a very long line. The same for other `fail:` modules with longer msg.

Reply:
Agree with you.